app/lib/views/navigator/waypoints.dart

```diff
@@ -15,28 +15,76 @@ class WaypointsView extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    const origin = Waypoint.origin;
     return BlocBuilder<TransformCubit, CameraTransform>(
       builder: (context, transform) {
         return BlocBuilder<DocumentBloc, DocumentState>(
           buildWhen: (previous, current) =>
               previous.page?.waypoints != current.page?.waypoints,
           builder: (context, state) {
             if (state is! DocumentLoadSuccess) return const SizedBox.shrink();
-            var waypoints = state.page.waypoints;
+            var waypoints = List.from(state.page.waypoints);
```

Comment:
```suggestion
            var waypoints = List<Waypoint>.from(state.page.waypoints);
```
I think this list needs to be typed

Reply:
Addressed in https://github.com/LinwoodDev/Butterfly/pull/817/commits/e3ab3035914d6dadc83da0f8e4d6cf49a68631fb